### DISCUSSION TITLE
so-rule getting error message

### DISCUSSION THREAD

**Discussion by BCordleRossVideo** [Security-Onion-Solutions/securityonion#10838] [2.4]
Hi - I'm running a fresh install of version 2.4.3.  I'm attempting to disable a SDI via command line, but I get an error when I run any of the so-rule commands.  

```
[*******@securityonion ~]$ sudo so-rule
(This should not happen, the system is in an error state if you see this message.)

More than one manager-type pillar exists, minion id's listed below:
  "/opt/so/saltstack/local/pillar/minions/adv_securityonion_standalone.sls", "/opt/so/saltstack/local/pillar/minions/securityonion_standalone.sls"
```
On-prem standalone deployment from SO ISO.  so-status showing all green.  sudo salt-call state.highstate came back with no errors.

Possibly related to discussion #10626.

(This is my first SO Discussion post, so feel free to chastise me if I've left out anything important or somehow broken protocol.)

**Comment by dougburks** [comment]
Have you tried disabling the SID via the new GUI in 2.4?
https://docs.securityonion.net/en/2.4/managing-alerts.html#disable-the-sid
https://docs.securityonion.net/en/2.4/administration.html#configuration

**Reply by BCordleRossVideo** [reply]
I've tried that now & it appears to be working!
